# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../strace/arm.patch
# Copyright (C) 2009 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- strace/linux/arm/syscallent.h	2008/12/29 20:04:15	1.17
+++ strace/ilnux/arm/syscallent.h	2009/01/01 23:20:38	1.18
@@ -431,6 +431,7 @@
 	{ 5,	0,	printargs,		"SYS_398"	}, /* 398 */
 	{ 5,	0,	printargs,		"SYS_399"	}, /* 399 */
 
+#ifndef __ARM_EABI__
 #if SYS_socket_subcall != 400
  #error fix me
 #endif
@@ -481,3 +482,4 @@
 	{ 4,	TI,	sys_shmdt,		"shmdt"		}, /* 440 */
 	{ 4,	TI,	sys_shmget,		"shmget"	}, /* 441 */
 	{ 4,	TI,	sys_shmctl,		"shmctl"	}, /* 442 */
+#endif
